packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css

```diff
@@ -148,7 +151,9 @@
 }
 
 .list-item:hover {
-  background: var(--color-primary);
+  background-color: var(--color-primary);
+  transition: all 0.25s ease-in-out;
```

Comment:
Perhaps we could apply this transition only to `background-color`. Seems like `border-radius` is also animated now and I don't know if that's expected.

Reply:
border radius should not be animated

Reply:
I'll remove transitions from this PR. It'll be cleaner then. And will work on them on a separate ones, also when @FilPob adds more details for them in design. 🙏